examples/example_template/example_template.py

```diff
@@ -87,3 +80,11 @@ def dummy():
 # code directly will see the plots; this is not necessary for creating the docs
 
 plt.show()
+
```

Comment:
And why has this block moved?

Reply:
To make sure it does not interrupt the flow of an example.